rust/agama-lib/share/profile.schema.json

```diff
@@ -869,9 +869,9 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
-        "reuse": {
+        "reuseIfPossible": {
```

Comment:
The document [auto_storage](https://github.com/openSUSE/agama/blob/master/doc/auto_storage.md) proposes to have a [reuse](https://github.com/openSUSE/agama/blob/master/doc/auto_storage.md#keeping-an-existing-file-system-or-encryption-layer) property, but *reuseIfPossible* was used instead because the following reasons:

* A file system cannot be reused if the partition is going to be created (*search* is not specified).
* A file system cannot be reused if the device is going to be encrypted (the encryption layer is not reused). 
* A file system cannot be reused if the device is not currently formatted.

In order to validate JSON config to the usage of *reuse* in the schema would make the schema more complex with very little benefit.  

TODO